features/step_definitions/asset_host_steps.rb

```diff
@@ -6,20 +6,16 @@
   Rails.configuration.three_scale.asset_host = cdn_url
 end
 
-Then /^((javascript|font)\s)?assets should be loaded from the asset host$/ do |asset_type|
+Then "(javascript )assets should be loaded from the asset host" do
```

Comment:
Why remove font loading test?

Reply:
Right now css is being "injected" in the DOM and this test asserts the font-family prop has the correct url. But after this upgrade, stylesheets are linked and downloaded as a separate source:
```html
<link rel="stylesheet" media="screen" href="http://provider-admin.3scale.localhost:3000/packs/css/provider.css">
```
So this assertion is not possible anymore.

Reply:
Shouldn't they still be downloaded from CDN?

Reply:
They are. But this test does not test that. It tests that this is present **in the DOM**:
```css
@font-face {
    font-family: "RedHatDisplay";
    src: url(/packs/static/@patternfly/patternfly/assets/fonts/RedHatDisplay/RedHatDisplay-Regular-0c3b86386764101597ac.eot);
    src: url(/packs/static/@patternfly/patternfly/assets/fonts/RedHatDisplay/RedHatDisplay-Regular-0c3b86386764101597ac.eot?#iefix) format("embedded-opentype"), url(/packs/static/@patternfly/patternfly/assets/fonts/RedHatDisplay/RedHatDisplay-Regular-7791e86cb49866c2b138.woff) format("woff");
...
```
Which isn't anymore. Now this css is downloaded as an external file (like js)